src/components/Blogs/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import SearchBar from "../SearchBar";
+import { Container } from "react-bootstrap";
+import "./index.scss";
+
+export default function Blogs() {
+  const [blogs, setBlogs] = useState([]);
+  const URL =
+    "https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts";
+  const fetchBlogs = async () => {
+    const data = await fetch(URL);
+    const fetchedBlogs = await data.json();
+    setBlogs(fetchedBlogs);
+    console.log(fetchedBlogs);
+  };
+  useEffect(() => {
+    fetchBlogs();
+  }, []);
+  const NUM_BLOGS_TO_SHOW = 9;
```

Comment:
consts with uppercase should be outside of function just below the import

Reply:
done